### ISSUE TITLE
package bindgen-cli not found

### ISSUE THREAD

**Issue by EriKWDev** [nabijaczleweli/cargo-update#206]
Crash on fresh install and usage

```
> RUST_BACKTRACE=1 cargo install-update -a
    Updating registry 'https://github.com/rust-lang/crates.io-index'

thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: "package bindgen-cli not found"', /home/erik/.cargo/registry/src/github.com-1ecc6299db9ec823/cargo-update-11.0.0/src/ops/mod.rs:216:149
stack backtrace:
   0: rust_begin_unwind
             at /rustc/897e37553bba8b42751c67658967889d11ecd120/library/std/src/panicking.rs:584:5
   1: core::panicking::panic_fmt
             at /rustc/897e37553bba8b42751c67658967889d11ecd120/library/core/src/panicking.rs:142:14
   2: core::result::unwrap_failed
             at /rustc/897e37553bba8b42751c67658967889d11ecd120/library/core/src/result.rs:1785:5
   3: cargo_update::ops::RegistryPackage::pull_version
   4: cargo_install_update::actual_main
   5: cargo_install_update::main
note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace
```
```
>  which bindgen
~/.cargo/bin/bindgen
> cargo search bindgen-cli
bindgen-cli = "0.61.0"                 # Automatically generates Rust FFI bindings to C and C++ libraries
...
> cargo --version
cargo 1.65.0 (4bc8f24d3 2022-10-20)
>  cargo install-update --version
cargo-install-update 11.0.0
```
Running on Linux, Fedora. Installed all dependencies listed.

**Comment by nabijaczleweli** [comment]
Looks like a repeat of #205, which is waiting on reporter to confirm the fix; can you try the current `master` branch (at least aa28f9164f5c92f63ae79d84bf2a5fccb2e19d31)?

**Comment by The-Mr-L** [comment]
I can confirm using the master doss fix the issue for a different package though  "flutter_rust_bridge_codegen"   :)

**Comment by nabijaczleweli** [comment]
Great, released in [`v11.1.0`](https://github.com/nabijaczleweli/cargo-update/releases/tag/v11.1.0)
